nixosConfigurations/services/steward.nix

```diff
@@ -33,18 +33,30 @@ with flake-utils.lib.system; let
     })
   ];
 
+  attest-nginx = {config, ...}: {
+    services.nginx.virtualHosts.${config.networking.fqdn} = {
```

Comment:
this belongs in `mkSteward`, since it's used by all Steward services.
Once the production instance stops using Nginx, we can guard this functionality by a boolean argument to `mkSteward`